src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
Ah ok, I see. But write ``define('E_SOMETHING_UNRELATED', 2)`` is strange or even unrecommended. If you want add your own ``E_*`` flag, you continue following the power of 2 by writting ``define('E_SOMETHING_UNRELATED', 32768)`` no?